db/migrate/20231107235634_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.integer :user_id
+      t.integer :post_id
+      t.text :text
+
+      t.timestamps
+    end
+  end
```

Comment:
- [ ] Good job so far creating the model for posts, but it would be nice if you could add references for the user and post that will create author_id and a corresponding index as required respectively. Let's fix that with the code below 👍🏽

👇🏾
- `t.references :user, foreign_key: true`
- `t.references :post, foreign_key: true`